draft-ietf-tls-subcerts.md

```diff
@@ -95,45 +97,64 @@ clarity, we will refer to the certificate issued by the CA as a "certificate",
 or "delegation certificate", and the one issued by the operator as a "delegated
 credential".
 
+## Change Log
+
+(\*) indicates changes to the wire protocol.
+
+draft-02
+
+  * Drop support for TLS 1.2.
+
+  * Add the protocol version and credential signature algorithm to the
+    Credential structure. (*)
+
+  * Specify undefined behavior in a few cases: when the client receives a DC
+    without indicated support; when the client indicates the extension in an
+    invalid protocol version; and when DCs are sent as extensions to
+    certificates other than the end-entity certificate.
+
+
 # Solution Overview
 
 A delegated credential is a digitally signed data structure with two semantic
-fields: a validity interval, and a public key (with its associated algorithm).
-The signature on the credential indicates a delegation from the certificate that
-is issued to the TLS server operator. The secret key used to sign a credential
-is presumed to be one whose corresponding public key is contained in an X.509
-certificate that associates one or more names to the credential.
+fields: a validity interval and a public key (along with its associated
+algorithm).  The signature on the credential indicates a delegation from the
+certificate that is issued to the TLS server operator.  The secret key used to
+sign a credential is presumed to be one whose corresponding public key is
+contained in an X.509 certificate that associates one or more names to the
+credential.
 
 A TLS handshake that uses credentials differs from a normal handshake in a few
 important ways:
 
 * The client provides an extension in its ClientHello that indicates support
   for this mechanism.
 * The server provides both the certificate chain terminating in its certificate
-  as well as the credential.
+  as well as the delegated credential.
 * The client uses information in the server's certificate to verify the
-  signature on the credential and verify that the server is asserting an
-  expected identity.
+  delegation and that the server is asserting an expected identity.
 * The client uses the public key in the credential as the server's
   working key for the TLS handshake.
 
-This document specifies the use of delegated credentials in TLS 1.3 or later;
-their use in prior versions of the protocol is explicitly disallowed.
+As detailed in {{delegated-credentials}}, the delegated credential is
+cryptographically bound to delegation certificate and the protocol in which the
```

Comment:
to `the` leaf certificate, i think leaf is less confusing than using delegated and delgation.

Reply:
Done.